olmo/model.py

```diff
@@ -1058,6 +1082,8 @@ def reset_parameters(self):
             wte_cutoff_factor = self.config.init_cutoff_factor or 3.0
         elif self.config.init_fn == InitFnType.full_megatron:
             wte_std = self.config.init_std
+            if self.config.scale_emb_init:
+                wte_std *= math.sqrt(self.config.d_model)
```

Comment:
Should we apply this to other init methods as well? The docstring for this option makes it seem like it would be applied to all of them.

Reply:
We had discussed this a bit at the time of https://github.com/allenai/OLMo/pull/607 , ultimately deciding that it was not something that was important to decide right away. I've added a clarification in the docstring.